### PR TITLE
[OSCI] Bug fix: Correct file path for importing Query component

### PR DESCRIPTION
**Pull request by JohnathonBowers**
### Description
This change clarifies the file path for the Query export in five files. Before the change, the OUI documentation site was not loading on my browser, and the console displayed the warnings in the following screenshot:

Screenshot 2023-10-05 at 5 06 46 PM

After the change, the documentation site loads successfully, and the warnings regarding the Query export are absent from my console:

Screenshot 2023-10-09 at 2 00 31 PM

### Issues Resolved
Fixes issue [#1063](https://github.com/opensearch-project/oui/issues/1063) 

### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] All tests pass
  - [x] `yarn lint`
  - [x] `yarn test-unit`
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by BSFishy — COMMENTED** [review]
Looks good on first pass, will update issue to get rid of the `Ast` issue

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 ### 🐛 Bug Fixes
 
 - Add exit code to compile-scss script on failure ([#1024](https://github.com/opensearch-project/oui/pull/1024))
+- Clarify file path for Query export ([#1069](https://github.com/opensearch-project/oui/pull/1069))
```
Comment (BigSamu) [inline CHANGELOG.md:19]: I would say instead, Correct file path for Query export ([#1069](https://github.com/opensearch-project/oui/pull/1069))

Comment (JohnathonBowers) [reply CHANGELOG.md:19]: Good point, Samuel. I will update the language.

**Review by BSFishy — APPROVED** [review]
Assuming the `AST` change is going in another PR, this looks good

**Comment by JohnathonBowers** [comment]
@BSFishy Yes, I'll open a separate issue for the Ast fix.

**Review by BigSamu — APPROVED** [review]
LGTM

**Review by Willie-The-Lord — COMMENTED** [review]
LGTM! This resolves the issue I was experiencing with my blank website (Mac).

**Review by joshuarrrr — APPROVED** [review]

**Pull request merged**